mathjax3-ts/output/chtml/fonts/tex.ts

```diff
@@ -80,103 +118,332 @@ export class TeXFont extends FontData {
+        return styles;
+    }
+
+    /*
+     * @param{StyleList} styles    The style object to add styles to
+     * @param{number} n            The unicode character number of the delimiter
+     * @param{DelimiterData} data  The data for the delimiter whose CSS is to be added
+     */
+    protected addDelimiterStyles(styles: StyleList, n: number, data: DelimiterData) {
+        if (!data.stretch) return;
+        if (data.dir === 'V') {
+            const c = this.char(n);
+            const Hb = this.addDelimiterVPart(styles, c, 'beg', data.stretch[0]);
+            this.addDelimiterVPart(styles, c, 'ext', data.stretch[1]);
+            const He = this.addDelimiterVPart(styles, c, 'end', data.stretch[2]);
+            const css: StyleData = {};
+            if (Hb) {
+                css['border-top-width'] = this.em(Math.max(0, Hb - .03));
+            }
+            if (He) {
+                css['border-bottom-width'] = this.em(Math.max(0, He - .03));
+                css['margin-bottom'] = this.em(-He);
+            }
+            styles['.MJX-TEX mjx-stretchy-v[c="' + c + '"] mjx-ext'] = css;
+        } else {
+            //  FIXME: still need to add this
+        }
+    }
+
+    /*
+     * @param{StyleList} styles  The style object to add styles to
+     * @param{string} c          The vertical character whose part is being added
+     * @param{string} part       The name of the part (beg, ext, end, mid) that is being added
+     * @param{number} n          The unicode character to use for the part
+     * @return{number}           The total height of the character
+     */
+    protected addDelimiterVPart(styles: StyleList, c: string, part: string, n: number) {
+        if (!n) return 0;
+        const data = this.getChar('normal', n) || this.getChar('-size4', n);
+        const css: StyleData = {content: '"' + this.char(n, true) + '"'};
+        if (part !== 'ext') {
+            css.padding = this.em(data[0]) + ' 0 ' + this.em(data[1]);
+        }
+        styles['.MJX-TEX mjx-stretchy-v[c="' + c + '"] mjx-' + part + ' mjx-c::before'] = css;
+        return data[0] + data[1];
+    }
+
+    /*
+     * @param{StyleList} styles  The style object to add styles to
+     * @param{string} vclass     The variant class string (e.g., .mjx-b) where this character is being defined
+     * @param{number} n          The unicode character being defined
+     * @param{CharData} data     The bounding box data and options for the character
+     */
+    protected addCharStyles(styles: StyleList, vclass: string, n: number, data: CharData) {
+        const [h, d, w, options] = data as [number, number, number, CharOptions];
+        const css: StyleData = {};
+        if (options.css) {
+            if (options.css & CSS.width)   css.width = this.em(w);
+            if (options.css & CSS.padding) css.padding = this.em(h) + ' 0 ' + this.em(d);
+            if (options.css & CSS.content) css.content = '"' + (options.c || this.char(n, true)) + '"';
```

Comment:
If styling.

Reply:
Ok, but it is harder to read what is happening there.  You can't see the symmetry of the three lines as clearly.

Forcing one-line if-then statements onto separate lines seems like a stupid rule to me.  Personally, I think it makes things more difficult to read.  (I removed it from the `tslint.config` some time ago -- you didn't complain.  :-)